src/Components/Item.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import "./Item.css"
+
+
+function Item({ todo, setTodos }) {
+
+    const [edit, setEdit] = useState(false)
+    const [item, setItem] = useState(todo.item);
+
+    const handleDelete = () => {
+        if (!todo.isComplete) {
+            setEdit(false)
+        }
+        setTodos((prevTodos) => [...prevTodos.map(todoItem => {
```

Comment:
unnecessary array and spread as map already return and array.

Reply:
spread operator removed

Reply:
fixed for handleEdit method but still there in handleDelete

Reply:
fixed for handleDelete also